rules-tests/Configs/Rector/ClassMethod/AddRouteAnnotationRector/config/configured_rule.php

```diff
@@ -3,11 +3,16 @@
 declare(strict_types=1);
 
 use Rector\Config\RectorConfig;
-use Rector\Symfony\Contract\Bridge\Symfony\Routing\SymfonyRoutesProviderInterface;
+
+use Rector\Symfony\Bridge\Symfony\Routing\SymfonyRoutesProvider;
 use Rector\Symfony\Configs\Rector\ClassMethod\AddRouteAnnotationRector;
+use Rector\Symfony\Contract\Bridge\Symfony\Routing\SymfonyRoutesProviderInterface;
 use Rector\Symfony\Tests\Configs\Rector\ClassMethod\AddRouteAnnotationRector\Source\DummySymfonyRoutesProvider;
 
 return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->singleton(SymfonyRoutesProvider::class);
+    $rectorConfig->alias(SymfonyRoutesProvider::class, SymfonyRoutesProviderInterface::class);
+
```

Comment:
Ooops, it should be added into the `config/sets` instad of `rules-tests` :)

Reply:
@TomasVotruba it seems `AddRouteAnnotationRector` is never exists in `SetList` so it manually added.

Reply:
@TomasVotruba what `SetList` that better to register `AddRouteAnnotationRector`? code-quality? or move the missing alias in rector-src's config?

Reply:
@hildebro as it registered manually instead of via `SetList`, you can register above config, the readme seems needs update.

Reply:
I created PR to update Readme for it:

- https://github.com/rectorphp/rector-symfony/pull/520